src/main/java/org/candlepin/subscriptions/subscription/SubscriptionSyncController.java

```diff
@@ -343,18 +335,47 @@ public void deleteSubscription(String subscriptionId) {
   @Transactional
   public void forceSyncSubscriptionsForOrg(String orgId) {
     var subscriptions = subscriptionService.getSubscriptionsByOrgId(orgId);
+
+    log.info("Start getting active subscriptions for org");
     var subscriptionMap =
         getActiveSubscriptionsForOrg(orgId)
             .collect(
                 Collectors.toMap(
                     org.candlepin.subscriptions.db.model.Subscription::getSubscriptionId,
                     sub -> sub));
+    log.info("Stop getting active subscriptions for org");
+
+    log.info("Start syncing subscriptions", subscriptionMap);
+    subscriptions.stream()
+        .filter(this::shouldSyncSub)
```

Comment:
shouldSyncSub() is called again in 'syncSubscription()' so I think we should get rid of it here.